tree_queries/compiler.py

```diff
@@ -184,7 +184,16 @@ def as_sql(self, *args, **kwargs):
     def get_converters(self, expressions):
         converters = super(TreeCompiler, self).get_converters(expressions)
         for i, expression in enumerate(expressions):
-            if any(f in str(expression) for f in ("tree_path", "tree_ordering")):
+            # We care about tree fields and annotations only
+            if not hasattr(expression, "sql"):
+                continue
+
+            if any(
+                [
+                    expression.sql == f
+                    for f in ("__tree.tree_path", "__tree.tree_ordering")
+                ]
+            ):
```

Comment:
That might work too?

```suggestion
            if expression.sql in {"__tree.tree_path", "__tree.tree_ordering"}:
```